iree_tests/configs/config_sdxl_scheduled_unet_gpu_rocm.json

```diff
@@ -0,0 +1,34 @@
+{
+    "config_name": "gpu_rocm",
+    "iree_compile_flags" : [
+      "--iree-hal-target-backends=rocm",
+      "--iree-rocm-target-chip=gfx90a",
+      "--iree-opt-const-eval=false",
+      "--iree-codegen-transform-dialect-library=specs/attention_and_matmul_spec.mlir",
+      "--iree-global-opt-propagate-transposes=true",
+      "--iree-opt-outer-dim-concat=true",
+      "--iree-vm-target-truncate-unsupported-floats",
+      "--iree-llvmgpu-enable-prefetch=true",
+      "--iree-opt-data-tiling=false",
+      "--iree-codegen-gpu-native-math-precision=true",
+      "--iree-codegen-llvmgpu-use-vector-distribution",
+      "--iree-preprocessing-pass-pipeline=builtin.module(iree-preprocessing-transpose-convolution-pipeline, util.func(iree-preprocessing-pad-to-intrinsics))"
+    ],
+    "iree_run_module_flags": [
+      "--device=hip",
+      "--parameters=model=real_weights.irpa",
+      "--module=sdxl_scheduled_unet_pipeline_fp16_rocm.vmfb",
+      "--input=1x4x128x128xf16=@inference_input.0.bin",
+      "--input=2x64x2048xf16=@inference_input.1.bin",
+      "--input=2x1280xf16=@inference_input.2.bin",
+      "--input=1xf16=@inference_input.3.bin",
+      "--expected_output=1x4x128x128xf16=@inference_output.0.bin",
+      "--expected_f16_threshold=0.8f"
+    ],
+    "skip_compile_tests": [],
+    "skip_run_tests": [],
+    "expected_compile_failures": [
+        "resnet50"
+    ],
```

Comment:
Let's drop resnet from this config, then only run pytest with this config using `sdxl` patterns